Formula/haruhi-dl.rb

```diff
@@ -0,0 +1,23 @@
+class HaruhiDl < Formula
+  desc "Fork of youtube-dl, focused on bringing a fast, steady stream of updates"
+  homepage "https://haruhi.download"
+  url "https://git.sakamoto.pl/laudompat/haruhi-dl/-/archive/v2021.03.21/haruhi-dl-v2021.03.21.tar.gz"
+  sha256 "8b7304635ecf7b6c7f36de9ae15433827e9690304f8f6e6f495805b9c820278e"
+  license "LGPL-3.0-or-later"
+
+  depends_on "pandoc" => :build
```

Comment:
This will keep it from building on ARM. Is this just for building documentation? If so, then we should skip those on ARM.